connector/datadogconnector/go.mod

```diff
@@ -3,8 +3,8 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/connector/datad
 go 1.21.0
 
 require (
-	github.com/DataDog/datadog-agent/pkg/proto v0.52.1-0.20240321095122-a3c5dbb936ae
-	github.com/DataDog/datadog-agent/pkg/trace v0.52.1-0.20240321095122-a3c5dbb936ae
+	github.com/DataDog/datadog-agent/pkg/proto v0.54.0-devel.0.20240419195358-8549c987746f
+	github.com/DataDog/datadog-agent/pkg/trace v0.54.0-devel.0.20240419195358-8549c987746f
```

Comment:
Is it necessary to change `pkg/trace` to a commit ?

Reply:
It's not necessary but I thought it would be safer to upgrade all datadog-agent modules to match the same version